wesnoth-units.po

```diff
@@ -5652,15 +5652,13 @@ msgstr ""
 
 #. [unit_type]: id=Fire Ant, race=monster
 #: data/core/units/monsters/Fire_Ant.cfg:11
-#, fuzzy
-#| msgid "Fire Dragon"
 msgid "Fire Ant"
-msgstr "Ohnivý drak"
+msgstr "Ohnivý obří mravenec"
```

Comment:
doufám, že máte důvod ho nafukovat

Reply:
1) Podle obrázku to není mravenec standartní velikosti
2) V Units je jednotka Obří mravenec (lvl 0) s nižšími staty. Sice na sebe tyto dva mravenci nenavazují (jsou to samostatné jednotky), ale přece jenom je ohnivý mravenec lvl 1 a s vyššími staty.
Z těchto důvodů jsem si dovolil ho přifouknout.

Reply:
Ale napadá mě tedy např. ještěr bažinný a medvěd jeskynní. Tyto upravím a další zvážím při dalším updatu.